.golangci.yml

```diff
@@ -1,19 +1,36 @@
 linters:
-  enable:
-    - gofmt
-    - govet
-    - errcheck
-    - staticcheck
-    - gosimple
-    - ineffassign
-    - typecheck
-    - unused
-    - misspell
-    - gocyclo
-    - gosec
-    - revive
+  enable-all: true
+  disable:
+    - exportloopref # Deprecated
+    - varnamelen # Not relevant
+    - exhaustruct # Not relevant
+    - ireturn # Not relevant
+    - nlreturn # Not relevant
+    - contextcheck # Too strict
```

Comment:
Should be enabled imo, it's often useful to let you know when you could be passing a context somewhere and aren't. In 99% of cases where you could pass a context, you should